app/app.go

```diff
@@ -48,7 +49,7 @@ type App struct {
 	mgr *lifecycle.Manager
 
 	db     *sql.DB
-	l      net.Listener
+	l      *multiListener
```

Comment:
This can remain `net.Listener` as `multiListener` should implement that interface.